contracts/standard/arbitration/MultipleArbitrableTransaction.sol

```diff
@@ -340,8 +340,13 @@ contract MultipleArbitrableTransaction {
         uint[] memory transactionIDsBigArr = new uint[](transactions.length);
         uint count = 0;
         for (uint i = 0; i < transactions.length; i++)
+        {
```

Comment:
Why did you add this?

Reply:
It was like this:
```
1        for (uint i = 0; i < transactions.length; i++)
2           if (transactions[i].seller == _address || transactions[i].buyer == _address)
3                transactionIDsBigArr[count++] = i;
```
Prettier gives an error:  
> 3:0      error      Only use indent of 8 spaces.  

Reducing indentation to 8 from 12 like this fixes the error:
```
1        for (uint i = 0; i < transactions.length; i++)
2           if (transactions[i].seller == _address || transactions[i].buyer == _address)
3           transactionIDsBigArr[count++] = i;
```

But this is a misleading indentation as line 3 should be one level more intended than line 2.

To fix it I added omitted braces.

Reply:
@ferittuncer 
The indentation was wrong.
You had 8, 11, and 16.
It should be 8, 12, and 16.

Reply:
In the comment? Yes. 

But I still can't pass Prettier using 8 12 16.

Reply:
I sent a commit to demonstrate.

Reply:
Yeah, this one works. So it seems the bug happens when braces omitted multiple levels.

I'll fix this and squash all the commits. @epiqueras